app/views/employees/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="container-fluid">
+  <div class="row p-3 bg-table">
+    <h3>Manage Employees</h3>
+    <div class="ml-auto">
```

Comment:
With the current implementation, the rendered employees here are hardcoded, pls ensure there is a followup ticket for creating the table in the DB and making it work with it 😄 

Or in case it would be part of the same ticket, I think is great having incremental PR's so I'm with merging it as is

Reply:
This ticket only specifies the front end, I think the next issue will be a CRUD for employees and then we'll remove the hardcoded values